blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -100,7 +100,7 @@ def set_op(self, device: Device, message: str) -> None:
         ManagerDeviceMenu.__ops__[device.get_object_path()] = message
         for inst in ManagerDeviceMenu.__instances__:
             if not hasattr(inst, "SelectedDevice"):
```

Comment:
I think this condition should be added to the next `if` with `or` to a) show the log message and b) avoid generating twice.